core/src/main/java/org/apache/calcite/rel/rules/ProjectToWindowRule.java

```diff
@@ -214,6 +215,11 @@ public ProjectToLogicalProjectAndWindowRule(
               return true;
             }
 
+            // TODO: Is this correct?
```

Comment:
Seems like it? I don't fully understand how this rule works.

Reply:
I also think it should be correct because its the same as a literal. Removing the TODO.